switchbot/devices/meter.py

```diff
@@ -1 +1,165 @@
-from __future__ import annotations
+from datetime import datetime
+
+from .device import SwitchbotDevice, SwitchbotOperationError
+
+
+class SwitchbotMeterProCO2(SwitchbotDevice):
+    """API to control Switchbot Meter Pro CO2."""
+
+    """
+    Command code to get/set the displayed time offset. You basically set the
+    offset whenever you manually set the device display time in the Switchbot
+    app.
+
+    The displayed time is calculated as the internal device time (usually comes
+    from the factory settings or set by the Switchbot app upon syncing)
+    + offset. The offset is provided in seconds and can be
+    positive or negative.
+    """
+    COMMAND_TIME_OFFSET = "570f680506"
```

Comment:
Please make them top level constants so they don’t become park of the public api

Reply:
Done! Also I reshaped the history a bit, because I found a couple of embarassing bugs during tests with the  device. That TZ handling is weird, especially when it comes to TZs with minute offsets.